File: android/worker/app/src/main/java/com/workforcex/worker/api/Notification.java
package com.workforcex.worker.api;

import com.google.gson.annotations.SerializedName;

public class Notification {
    @SerializedName("id")
    public String id;

    @SerializedName("message")
    public String message;

    @SerializedName("read")
    public boolean isRead;

    @SerializedName("createdAt")
    public String createdAt;

    @SerializedName("linkType")
    public String linkType;

    @SerializedName("linkId")
    public String linkId;
}

File: android/employer/shared_employer/src/commonMain/kotlin/com/workforcex/shared_employer/models/Notification.kt
package com.workforcex.shared_employer.models

import com.google.gson.annotations.SerializedName

data class Notification(
    @SerializedName("id")
    val id: String,

    @SerializedName("message")
    val message: String,

    @SerializedName("read")
    val isRead: Boolean,

    @SerializedName("createdAt")
    val createdAt: String,

    @SerializedName("linkType")
    val linkType: String,

    @SerializedName("linkId")
    val linkId: String,

    @SerializedName("jobTitle")
    val jobTitle: String
)